### PR TITLE
chore: improve model not found error msg

### PR DESCRIPTION
**Pull request by ZiniuYu**
We did not list all available models when the user input a wrong model name in torch runtime previously, this pr fix this
this pr also reformat the error msg, it now looks like

image

### CODE REVIEW

**Review by numb3r3 — CHANGES_REQUESTED** [review]
What happens on other backend, `ONNX` and `TensorRT`

File: server/clip_server/model/clip_model.py [hunk 1]
```diff
@@ -35,7 +35,9 @@ def __new__(cls, name: str, **kwargs):
 
                 instance = super().__new__(MultilingualCLIPModel)
             else:
-                raise ValueError(f'The CLIP model name=`{name}` is not supported.')
+                raise ValueError(
+                    f'Model {name} not found; available models = {list(_OPENCLIP_MODELS.keys()) + list(_MULTILINGUALCLIP_MODELS.keys())} '
```
Comment (numb3r3) [inline server/clip_server/model/clip_model.py:39]: I would prefer to use `CLIP model name="{name}" ...` to highlight the name

**Comment by ZiniuYu** [comment]
> What happens on other backend, `ONNX` and `TensorRT`

the are fine

File: server/clip_server/model/clip_model.py (hunk 1 above)
Comment (numb3r3) [reply server/clip_server/model/clip_model.py:39]: And what's more, we can improve the display by using rich table (or other api), e.g., 
```
- RN50
- RN16
- .....
```

**Review by numb3r3 — CHANGES_REQUESTED** [review]

**Review by numb3r3 — APPROVED** [review]

**Pull request merged**